modules/ref_runtimes_metering_labels.adoc

```diff
@@ -15,4 +15,12 @@ You can add metering labels to your {ProductName} pods and check Red Hat subscri
 * `com.redhat.product-name: {product-name}`
 * `com.redhat.product-version: {product-version}`
 
-See link:{metering-doc-root}[Configuring and using Metering in OpenShift Container Platform] for more information.
+[NOTE]
+====
+If you use an operator to deploy and manage {ProductName} pods, you should not manually add metering labels to those pods.
```

Comment:
```suggestion
If you use an operator to deploy and manage {ProductName} pods, do not manually add metering labels to those pods.
```

Reply:
@oraNod The original wording is OK, just a small suggestion. Feel free to disregard at your discretion:)

Reply:
@inoxx03 Thanks, Stefan. After a bit of discussion with Nicole and some more thought I'm going to make another change but it's pretty trivial. I'll keep you posted. And thanks for the review. Enjoy your PTO tomorrow.